### PR TITLE
Automatically determine num_actions and num_chance_outcomes in stochastic_muzero_policy

### PR DESCRIPTION
**Pull request by carlosgmartin**
#71

### CODE REVIEW

File: mctx/_src/policies.py [hunk 1]
```diff
@@ -293,6 +289,11 @@ def stochastic_muzero_policy(
     search tree.
   """
 
+  num_actions = root.prior_logits.shape[-1]
+  num_chance_outcomes = decision_recurrent_fn(
```
Comment (fidlej) [inline mctx/_src/policies.py]: You can get the num_chance_outcomes from the existing decision_recurrent_fn call, at line 312.

**Review by fidlej — COMMENTED** [review]
Thanks.

**Review by fidlej — CHANGES_REQUESTED** [review]
Please, update the usage in policies_test.py.

**Comment by carlosgmartin** [comment]
@fidlej Done.

File: mctx/_src/policies.py [hunk 2]
```diff
@@ -309,9 +307,10 @@ def stochastic_muzero_policy(
   # construct a dummy afterstate embedding
   batch_size = jax.tree_util.tree_leaves(root.embedding)[0].shape[0]
   dummy_action = jnp.zeros([batch_size], dtype=jnp.int32)
-  _, dummy_afterstate_embedding = decision_recurrent_fn(params, rng_key,
+  output, dummy_afterstate_embedding = decision_recurrent_fn(params, rng_key,
```
Comment (fidlej) [inline mctx/_src/policies.py]: You can call it `dummy_output`.
And update the indentation to match the "(".
Sorry for being a perfectionist.

File: mctx/_src/policies.py [hunk 3]
```diff
@@ -309,9 +307,9 @@ def stochastic_muzero_policy(
   # construct a dummy afterstate embedding
   batch_size = jax.tree_util.tree_leaves(root.embedding)[0].shape[0]
   dummy_action = jnp.zeros([batch_size], dtype=jnp.int32)
-  _, dummy_afterstate_embedding = decision_recurrent_fn(params, rng_key,
-                                                        dummy_action,
-                                                        root.embedding)
+  dummy_output, dummy_afterstate_embedding = decision_recurrent_fn(
+    params, rng_key, dummy_action, root.embedding)
```
Comment (fidlej) [inline mctx/_src/policies.py]: Here, the identation should be 4 spaces, because it is a "hanging" indent.
Thanks.

**Pull request merged**